### PR TITLE
Improvements in REMReM documentation(eiffel-REMReM-generate)

### PR DESCRIPTION
**Pull request by Xlakdas**
Improvements in REMReM documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -336,14 +348,63 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 }' http://localhost:8080/eiffelsemantics?msgType=eiffelactivityfinished</pre>
 
 <p><strong>NOTE:</strong> For other than eiffelsemantics protocol, provide the Java opts as: </p>
-<pre>set JAVA_OPTS="-Djava.ext.dirs=/path/to/other/protocol/jars" in catalina.sh</pre>
-
+<pre>set JAVA_OPTS="-Djava.ext.dirs=/usr/home/jars/" in catalina.sh</pre>
```
Comment (k-svard-ericsson) [inline index.html]: Isn't it better to have an example path like "/path/to/jars" to make it obvious the user have to provide there own path?

File: index.html [hunk 2]
```diff
@@ -239,7 +238,11 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 <p>Gives:</p>
 
 <pre>[{"data":{"outcome":{"conclusion":"TIMED_OUT","description":"Compilation timed out."},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},{"name":"otherLog","uri":"isbn:0-486-27557-4"}]},"links":{"activityExecution":"aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","flowContext":"flowContext","causes":["cause1","cause2"]},"meta":{"domainId":"example.domain","id":"babcdb18-b3c8-46c0-8169-684733bf5ac9","type":"eiffelactivityfinished","version":"0.1.6-PBN2","time":1477033634382,"tags":["tag1","tag2"],"source":{"host":"host","name":"name","uri":"http://java.sun.com/j2se/1.3/","serializer":{"groupId":"G","artifactId":"A","version":"V"}}}}]</pre>
-
+<p><strong>For loading protocol jars other than eiffelsemantics :</strong></p>
+we need to provide the <strong><em>-Djava.ext.dirs="Path to other protocol jars".</em></strong> For example:
```
Comment (k-svard-ericsson) [inline index.html]: This line can be removed to be consistent with the documentation for publish

File: index.html [hunk 3]
```diff
@@ -239,7 +238,11 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
...
+<pre><code> $ java -Djava.ext.dirs="/usr/home/jars/" -jar generate-cli.jar -f eiffeljobfinished.json -t eiffeljobfinished -mp protocoltype
```
Comment (k-svard-ericsson) [inline index.html]: Isn't it better to have an example path like "/path/to/jars" to make it obvious the user have to provide there own path?

Comment (SantoshNC68) [inline index.html]: We had the example as /path/to/jars but the users were confused so we have added this, as an example in the documentation

**Pull request merged**